build

```diff
@@ -58,11 +58,6 @@ container_mount_opts=(
         -v "$PWD/$target_dir:/builder/.build"
 )
 
-if [ "$container_image" = localhost/builder ]; then
```

Comment:
This part of the build script is intended to speed up local dev / testing, by allowing to automatically rebuild the builder if any local changes were made.
It works if instead of downloading a release build script and placing it in the config directory, you instead sym link to the script inside a local copy of the builder repo.
We should not remove this

Reply:
we should probably document how this local dev / testing setup works though 😅

Reply:
but how do I use a locally built builder in a separate repo, that holds my features?
When I set the container image in the other repo to `localhost/builder` it tries to build the builder. Instead, it should use the builder that I have built before using `docker build -t localhost/builder .`

Reply:
I don't want to build the builder in the same repo that hold my features. I want to have it separated.

Reply:
The idea is to use a symlink to the build script inside the builder repo instead of a copy of the build script when doing local dev on the builder. That way the builder and the features are still separated, but the builder automatically detects that it is running in local dev mode and rebuilds itself iff necessary.